pyrene/src/utils/DateUtils.ts

```diff
@@ -0,0 +1,258 @@
+import isExists from 'date-fns/isExists';
```

Comment:
The idea is to move all functions that have to do with date-fns library inside this file, so that if we ever have to replace the date-fns library it can be easily done from one place. This approach was chosen because prior to the start of the DateTime component, the team had the task of removing momentJS from everywhere that it was used in Pyrene. Currently date-fns is also being used in the CalendarDateSelector component, TimeRangeSelector component and probably in other places. So the idea was that once this component is merged, all other components will call their date-fns functions from this utils file .